### ISSUE TITLE
Hyperparameters for pre-training GPT-2-XL

### ISSUE THREAD

**Issue by Arvid-pku** [karpathy/nanoGPT#389]
Thank you for your outstanding work, which helped me a lot. 
Can you please provide **the hyperparameters for pre-training GPT-2-XL**? 
I have trained using the hyperparameters of gpt2-medium, but the LOSS jumps around 30k steps. 
Looking forward to your reply!

**Comment by VatsaDev** [comment]
Only OpenAI trained XL, there isnt a open version, so youre hyperparams are googling.

Whats your per epoch steps? How many epochs are you training for?

**Comment by Arvid-pku** [comment]
Sorry for the late reply. We have solved the problem. (final loss: 2.47)
There's no 'epoch' in our setting. There are 0.4M tokens in one step, and 100k steps totally.
But the final performance is still not very good. Perhaps the training data and model arch are not very advanced.
